layers/meta-resin-edison/recipes-core/images/resin-image-initramfs.bbappend

```diff
@@ -0,0 +1 @@
+IMAGE_ROOTFS_MAXSIZE = "16384"
```

Comment:
And besides my last comment, from my knowledge, nobody tested resinHUP on the edison. So the status quo of resinHUP on edison is that it is not working until known otherwise